domain/model/state/state.go

```diff
@@ -119,16 +122,28 @@ SELECT uuid
 FROM cloud
 WHERE name = ?
 `
-	var (
-		cloudUUID string
-	)
+	var cloudUUID string
 	err := tx.QueryRowContext(ctx, cloudStmt, input.Cloud).Scan(&cloudUUID)
 	if errors.Is(err, sql.ErrNoRows) {
 		return fmt.Errorf("%w cloud %q", errors.NotFound, input.Cloud)
 	} else if err != nil {
 		return fmt.Errorf("getting cloud %q uuid: %w", input.Cloud, err)
 	}
 
+	userStmt := `
+SELECT uuid
+FROM user
+WHERE uuid = ?
+AND removed = false
+`
+	var userUUID string
+	err = tx.QueryRowContext(ctx, userStmt, input.Owner).Scan(&userUUID)
```

Comment:
It was done before this PR, but we as a project should be using sqlair

Reply:
yep i'll move it but in another PR. Separation of concerns.